### ISSUE TITLE
Documentation Page Design

### ISSUE THREAD

**Issue by Angelo-Ufelle** [zuri-training/auth_wiki_team_1_project#87]
Design a user interface for the Documentation Page

**Comment by jben2122** [comment]
I was assigned the documentation page to design for desktop view this i did in compliance to the the style guide, spacing system and the entire design system. this work was done together with (Caleb Stanly),and we took our time to visit several websites with documentation page, examined the pages and than as well came up with our own unique design which can be viewed below. I alsod created the Lo-Fi for the Documentation

Link to Hi-Fi Figma File: https://www.figma.com/file/t4QbYjbcsnhtRYHLUN9AIh/Team-1-Figma-Board?node-id=1566%3A18749

Link to Lo-Fi:

**Comment by Stanleycaleb** [comment]
The documentation page of the project was designed by myself and @jben2122. We worked together by browsing through templates of documentation pages, and then we worked on the different sections of the pages.
Attached to this is a link to the Lo-Fi and Hi-Fi of the web UI of the pages.

Hi-Fi: https://www.figma.com/file/t4QbYjbcsnhtRYHLUN9AIh/Team-1-Figma-Board?node-id=1566%3A18749
Lo-Fi: https://www.figma.com/file/t4QbYjbcsnhtRYHLUN9AIh/Team-1-Figma-Board?node-id=1591%3A33328
